_posts/en/newsletters/2024-05-01-newsletter.md

```diff
@@ -0,0 +1,243 @@
+  option in a way that wasn't documented.  [Bitcoin Core #22087][] made
+  configuration option parsing more strict, breaking the undocumented
+  support in Bitcoin Core 27.0, which [affected LND][gugger zmq] and
+  possibly other programs.  This PR makes the option officially
+  supported and slightly changes its semantics to make it consistent
+  with other options for Unix sockets in Bitcoin Core, such as the
+  change described in [Newsletter #294][news294 sockets].
+
+- [Core Lightning #7240][] adds support for retrieving required blocks
+  from the Bitcoin P2P network if the local Bitcoin node has pruned
+  them.  If the CLN node needs a block that has been pruned by its local
+  `bitcoind`, it will call the Bitcoin Core RPC `getblockfrompeer` to
+  request it from a peer.  If the block is successfully retrieved,
+  Bitcoin Core authenticates it by connecting it to the header
+  it keeps even for pruned blocks, and saves it locally, where it
+  can be retrieved using standard block-retrieval RPCs.
+
+- [Eclair #2851][] begins depending on Bitcoin Core 26.1 or greater and
+  removes code for ancestor-aware funding.  Instead, the upgrade allows
+  it to use Bitcoin Core's new native code for compensating for any _fee
+  deficit_ (see [Newsletter #269][news269 fee deficit]).
+
+- [LND #8147][], [#8422][lnd #8422], [#8423][lnd #8423], [#8148][lnd
+  #8148], [#8667][lnd #8667], and [#8674][lnd #8674] replace LND's old
+  sweeper with a new implementation, allowing the broadcast of
+  settlement transactions and any transactions needed to effectively fee
+  bump them.  Both the old and new implementations mostly accept the
+  same parameters, such as the deadline by which the transaction must be
+  confirmed and the starting feerate to use, with the new implementation
+  also adding a `budget` that is the maximum amount to pay in fees.  The
+  new implementation provides more configurability, makes writing tests
+  easier, makes use of both [CPFP][topic cpfp] and [RBF][topic rbf] fee
+  bumping (each when appropriate), does a better job of batching fee
+  bumps to save on fees, and only updates feerates each block rather
+  than every 30 seconds.
+
+- [LND #8627][] now defaults to rejecting user-requested changes to
+  channel settings that require above-zero _inbound forwarding fees_.  For
+  example, imagine Alice wants to forward a payment through Bob to
+  Carol.  By default, Bob can no longer use the newly-added LND feature
+  for inbound forwarding fees (see [Newsletter #297][news297 inbound])
+  to require that Alice pay extra.  This new default ensures that Bob's
+  node remains compatible with nodes that don't support inbound
+  forwarding fees (which is currently almost all LN nodes).  Bob can
+  choose to be backwards incompatible by overriding the default with the
+  `accept-positive-inbound-fees` LND configuration setting, or he can
+  possibly achieve the desired result while remaining backwards
```

Comment:
unsure on this suggestion
```suggestion
  achieve the desired result while remaining backwards
```

Reply:
Keeping as-is.  It's a possibility because achieving the desired result depends on other parties understanding negative fees.  If nobody except Bob knew about negative fees, then everyone would choose the same paths regardless of what negative fee he set.  With positive fees, ignorance would prevent payments from being forwarded, so it's not a hypothetical.